src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+<!--
+Guideline: This page is common to all SDKs; add the name of the SDK you're documenting as well as the code samples as instructed. If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can quickly complete an ideal integration.**
+-->
+
+Add Sentry's <!--SDK name--> SDK to automatically report errors, exceptions, and rejections in your application. 
```

Comment:
rejections doesnt mean anything to me, call it errors and crashes or just errors. I think we're trying to be too complex.